### PR TITLE
DOC-10062 clarify why SQL activity page % of runtime column does not equal 100%

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10062

In statements-table.md include file, added note to explain how % of All Runtime values may not sum to 100%.

Rendered preview: [% of All Runtime description](https://deploy-preview-18459--cockroachdb-docs.netlify.app/docs/v23.2/ui-statements-page.html#percent-of-all-runtime)

### CODE REVIEW

**Review by nkodali — APPROVED** [review]

File: src/current/_includes/v23.1/ui/statements-table.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Execution Count | Cumulative number of executions of statements with this finger
...
+<a id="percent-of-all-runtime"></a>% of All Runtime | How much time this statement fingerprint took to execute in proportion to all other statements (even those not displayed) that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.<br><br>NOTE: The sum of the values displayed for this column may not equal 100%. The percentage for each fingerprint is calculated by dividing the numerator consisting of the fingerprint's runtime by the denominator consisting of the sum of the runtimes for all statement fingerprints in the time interval. By “all statements fingerprints”, this means all user statement fingerprints (not only the displayed top <i>k</i> fingerprints based on the [search criteria](#search-criteria), but also statements executed within the time period not included in the displayed result set), as well as internal statement fingerprints that are never included in the displayed result set. The search criteria is applied subsequent to the calculation of `% of All Runtime`.
```
Comment (taroface) [inline src/current/_includes/v23.1/ui/statements-table.md:14]:
```suggestion
<a id="percent-of-all-runtime"></a>% of All Runtime | The percentage of execution time taken by this statement fingerprint compared to all other statements executed within the time period, including those not displayed. Runtime is calculated as the mean execution latency multiplied by the execution count.<br><br><b>Note:</b> The sum of the values in this column may not equal 100%. Each fingerprint's percentage is calculated by dividing the fingerprint's runtime by the sum of the runtimes for all statement fingerprints in the time interval. "All statement fingerprints" means all user statement fingerprints (not only those displayed by the [search criteria](#search-criteria), which are applied after the `& of All Runtime` calculation), as well as internal statement fingerprints not included in the displayed result set.
```

File: src/current/_includes/v23.2/ui/statements-table.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Execution Count | Cumulative number of executions of statements with this finger
...
+<a id="percent-of-all-runtime"></a>% of All Runtime | How much time this statement fingerprint took to execute in proportion to all other statements (even those not displayed) that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.<br><br>NOTE: The sum of the values displayed for this column may not equal 100%. The percentage for each fingerprint is calculated by dividing the numerator consisting of the fingerprint's runtime by the denominator consisting of the sum of the runtimes for all statement fingerprints in the time interval. By “all statements fingerprints”, this means all user statement fingerprints (not only the displayed top <i>k</i> fingerprints based on the [search criteria](#search-criteria), but also statements executed within the time period not included in the displayed result set), as well as internal statement fingerprints that are never included in the displayed result set. The search criteria is applied subsequent to the calculation of `% of All Runtime`.
```
Comment (taroface) [inline src/current/_includes/v23.2/ui/statements-table.md:14]: Same suggestion as earlier

File: src/current/_includes/v24.1/ui/statements-table.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Execution Count | Cumulative number of executions of statements with this finger
...
+<a id="percent-of-all-runtime"></a>% of All Runtime | How much time this statement fingerprint took to execute in proportion to all other statements (even those not displayed) that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.<br><br>NOTE: The sum of the values displayed for this column may not equal 100%. The percentage for each fingerprint is calculated by dividing the numerator consisting of the fingerprint's runtime by the denominator consisting of the sum of the runtimes for all statement fingerprints in the time interval. By “all statements fingerprints”, this means all user statement fingerprints (not only the displayed top <i>k</i> fingerprints based on the [search criteria](#search-criteria), but also statements executed within the time period not included in the displayed result set), as well as internal statement fingerprints that are never included in the displayed result set. The search criteria is applied subsequent to the calculation of `% of All Runtime`.
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/statements-table.md:14]: Same suggestion as earlier

**Review by taroface — CHANGES_REQUESTED** [review]
I found the description a bit hard to follow. Here's a suggestion, but I'm not sure if it is still accurate!

File: src/current/_includes/v23.1/ui/statements-table.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v23.1/ui/statements-table.md]: Incorporated some suggestions, kept some original sentences to maintain accuracy of explanation.

File: src/current/_includes/v23.2/ui/statements-table.md (hunk 2 above)
Comment (florence-crl) [reply src/current/_includes/v23.2/ui/statements-table.md]: Incorporated some suggestions, kept some original sentences to maintain accuracy of explanation.

File: src/current/_includes/v24.1/ui/statements-table.md (hunk 3 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/statements-table.md]: Incorporated some suggestions, kept some original sentences to maintain accuracy of explanation.

**Review by florence-crl — COMMENTED** [review]
TFTRs!

File: src/current/_includes/v23.1/ui/statements-table.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Execution Count | Cumulative number of executions of statements with this finger
...
-% of All Runtime | How much time this statement fingerprint took to execute compared to all other statements that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.
+<a id="percent-of-all-runtime"></a>% of All Runtime | The percentage of execution time taken by this statement fingerprint compared to all other statements executed within the time period, including those not displayed. Runtime is calculated as the mean execution latency multiplied by the execution count.<br><br><b>Note:</b> The sum of the values in this column may not equal 100%. Each fingerprint's percentage is calculated by dividing the fingerprint's runtime by the sum of the runtimes for all statement fingerprints in the time interval. "All statement fingerprints" means all user statement fingerprints (not only those displayed by the [search criteria](#search-criteria)), as well as internal statement fingerprints which are never included in the displayed result set. The search criteria is applied after the `% of All Runtime` calculation.
```
Comment (taroface) [inline src/current/_includes/v23.1/ui/statements-table.md:14]:
```suggestion
<a id="percent-of-all-runtime"></a>% of All Runtime | The percentage of execution time taken by this statement fingerprint compared to all other statements executed within the time period, including those not displayed. Runtime is calculated as the mean execution latency multiplied by the execution count.<br><br><b>Note:</b> The sum of the values in this column may not equal 100%. Each fingerprint's percentage is calculated by dividing the fingerprint's runtime by the sum of the runtimes for all statement fingerprints in the time interval. "All statement fingerprints" means all user statement fingerprints (not only those displayed by the [search criteria](#search-criteria)), as well as internal statement fingerprints that are never included in the displayed result set. The search criteria are applied after the `% of All Runtime` calculation.
```

**Review by taroface — APPROVED** [review]
LGTM! Just two small tweaks.

File: src/current/_includes/v23.1/ui/statements-table.md (hunk 4 above)
Comment (florence-crl) [reply src/current/_includes/v23.1/ui/statements-table.md:14]: good catches! tftr!
